tests/unit/cli/test_flows.py

```
@@ -0,0 +1,228 @@
+
+        mock_result = MagicMock()
+        mock_result.judgments = judgments
+        mock_result.accepted = False
+
+        with (
+            patch.object(flows, "run_review_tui", return_value=mock_result),
+            patch.object(flows, "display_review_results") as mock_display,
+        ):
+            flows.review(console, judgments, changes)
+
+        mock_display.assert_called_once_with(console, judgments, changes, False)
```

Ruff (0.14.7)

Boolean positional value in function call

(FBT003)